test/unit/runner.spec.js

```diff
@@ -119,9 +125,12 @@ describe('Runner', function () {
     });
 
     it('should not fail when a new common global is introduced', function () {
+      if (process.browser) {
```

Comment:
..no, actually, we don't want this to run in *any* browser, regardless if it's ES5-spec-compliant.